HighscoreWindow.h

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2022, Harshit Sharma <harshits908@gmail.com>
+ * Copyright 2023, Johan Wagenheim
+ * All rights reserved. Distributed under the terms of the MIT license.
+ */
+
+#ifndef HIGHSCORE_WINDOW_H
+#define HIGHSCORE_WINDOW_H
+
+
+#include <Window.h>
+#include <Messenger.h>
+
+class BStringView;
+class BTextControl;
+
+enum
+{
+	H2048_NAME_ENTERED = '48NE',
+};
+
+class HighscoreWindow : public BWindow{
```

Comment:
following style in Game.h
```suggestion
class HighscoreWindow : public BWindow
{
```